OpenSim/Simulation/Manager/Manager.h

```diff
@@ -95,38 +95,29 @@ class OSIMSIMULATION_API Manager
     double _ti;
     /** Final time of the simulation. */
     double _tf;
-    /** First dt in an integration. */
-    double _firstDT;
     
     /** Storage for the states. */
     std::unique_ptr<Storage> _stateStore;
 
-   int _steps;
-   /** Number of integration step tries. */
-   int _trys;
-   /** Maximum number of steps in an integration. */
-   int _maxSteps;
-   /** Flag for signaling a desired halt. */
-   bool _halt;
-   /** Minimum step size. */
-   double _dtMin;
```

Comment:
@carmichaelong I believe it is a bug that the `ForwardTool` is not respecting `_minDT` since it derives from `AbstractTool` and exposes the property in XML. We should probably add that fix (here or separately).